python/mxnet/test_utils.py

```diff
@@ -1920,7 +1919,7 @@ def chi_square_check(generator, buckets, probs, nsamples=1000000):
     _, p = ss.chisquare(f_obs=obs_freq, f_exp=expected_freq)
     return p, obs_freq, expected_freq
 
-def verify_generator(generator, buckets, probs, nsamples=1000000, nrepeat=5, success_rate=0.15):
+def verify_generator(generator, buckets, probs, nsamples=1000000, nrepeat=5, success_rate=0.25, alpha=0.05):
```

Comment:
How are we deciding on the success rate to be used?

Reply:
Success rate as defined in the function definition comment to be default to 25%
However, in the parameters it was wrongly set to 0.15. So I corrected it to 0.25
As far as alpha goes, same it is defined in the comment to be default to 0.05